jpsreport/methods/Method_I.cpp

```diff
@@ -294,7 +294,7 @@ void Method_I::GetIndividualFD(
 
 void Method_I::SetCalculateIndividualFD(bool individualFD)
 {
-    _calcIndividualFD = true;
+    _calcIndividualFD = individualFD;
```

Comment:
I haven't had any warning yet. I had chosen this setting because I need the output IndividualFD for method I. The true/false option is not required.

We can remove this unnecessary code in another PR.

Reply:
Alright! I removed the commit.